static/src/stylesheets/module/experiments/_acquisitions-epic-design-variations.scss

```diff
@@ -67,7 +67,8 @@
 // ============
 
 .contributions__paragraph--highlight {
-    background-color: transparentize($multimedia-main-2, .6);
+    background-color: #ffe399;
+    padding: 4px;
```

Comment:
that's true but it also looked a bit odd with the highlight exactly on the start of the characters, I have instead added 2px horizontal padding and adjusted the margins to keep the spacing the same as it would be without the highlight